src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/features.yml

```diff
@@ -1,12 +1,29 @@
 admin_features_index:
   path: /
-  methods: [ GET, POST ]
+  methods: [ GET ]
   defaults:
     _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\FeatureController::indexAction
     _legacy_controller: AdminFeatures
     _legacy_link: AdminFeatures
     _legacy_feature_flag: feature
 
+admin_features_horizontal_index:
+  path: /horizontal
+  methods: [ GET, POST ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\HorizontalFeatureController::indexAction
+    _legacy_controller: AdminFeatures
+
+admin_features_search:
+  path: /
+  methods: [ POST ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\CommonController::searchGridAction
+    _legacy_controller: AdminFeatures
+    _legacy_link: AdminFeatures
```

Comment:
```suggestion
    _legacy_link: AdminFeatures
    _legacy_feature_flag: feature
```

Reply:
https://github.com/PrestaShop/PrestaShop/pull/31689/commits/2c00425fa7e620edec854885106b3138722b349a
related to #31823